akka-remote/src/test/scala/akka/remote/transport/SystemMessageDeliveryStressTest.scala

```diff
@@ -97,36 +114,64 @@ abstract class SystemMessageDeliveryStressTest(msg: String, cfg: String)
   with DefaultTimeout {
   import SystemMessageDeliveryStressTest._
 
```

Comment:
please increase the `expectedTestDuration` to 120s, this test (sometimes) takes longer than default 60s